### PR TITLE
Simulation Time Resolution Notification

### LINKED ISSUE #69: Simulation time resolution
Stefan reports that student mostly crashed the nest-server when the sim resolution is smaller than 0.1.
Maybe we should consider a smaller range for simulation time resolution, e.g. 0.1, 1.0, 10.0.

### PR DESCRIPTION
**Pull request by jens-br**
Addressing #69, his PR adds a notification when small resolution values are selected. I think we should not forbid these values, in case a user needs them, but instead give a warning, so that the user is aware of that risk.

### CODE REVIEW

File: src/components/parameter/ParameterEdit.vue [hunk 1]
```diff
@@ -124,19 +124,30 @@
...
+                    :value="state.value"
+                    v-bind="attrs"
+                    v-on="on"
+                  />
+                </template>
+                <span>
+                  <v-icon v-text="'mdi-alert-outline'" color="white" />
+                  Beware: Small resolution values produce many datapoints, which
+                  can cause a high system load and thus freezes and lags!</span
+                >
```
Comment (babsey) [inline src/components/parameter/ParameterEdit.vue:149]:
```suggestion
                  Beware: Small resolution values produce many datapoints,
                  <br>
                  which might cause a high system load and thus freezes and lags!
                </span>          
```

**Review by babsey — COMMENTED** [review]
LGTM except small suggestions (See below).

File: src/components/parameter/ParameterEdit.vue (hunk 1 above)
Comment (jens-br) [reply src/components/parameter/ParameterEdit.vue]: Good point, I added it with the commit below.

**Comment by jens-br** [comment]
An open problem is that all watchers fire, so that the tooltip is shown for fractions of seconds. This seems to be a bit more complicated to solve.

**Comment by babsey** [comment]
LGTM.

**Pull request merged**